react-backend/client/src/components/authentication/logincb.js

```diff
@@ -0,0 +1,86 @@
+import React, { Component } from 'react';
+import { Input } from '../visitors/input';
+import { Button } from '../visitors/button';
+
+class CBlogin extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      email: '',
+      password: '',
+      error: [],
+    };
+  }
+
+  handleChange = e => {
+    let newState = {};
+    newState[e.target.name] = e.target.value;
+    this.setState(newState);
+  };
+
+  setError(messagesArray) {
+    this.setState({ error: messagesArray });
+  }
+
+  handleSubmit = e => {
+    e.preventDefault();
+
+    const checkData = {
+      formEmail: this.state.email,
+      formPswd: this.state.password,
+    };
+
+    fetch('/checkCBlogin', {
+      method: 'POST',
+      body: JSON.stringify(checkData),
+    })
+      .then(res => res.text())
+      .then(data => {
+        const EMAIL_ERROR = (
```

Comment:
since you use the same error info form `EMAIL_ERROR` here and in `signupcb` (and some other errors I think) it could be worth making something like an `errors.js` file and exporting all of them from there so that they are reusable.